source/MRViewer/MRRibbonFontManager.cpp

```diff
@@ -8,25 +8,55 @@
 #include "MRViewer.h"
 #include "MRRibbonMenu.h"
 #include "MRPch/MRSpdlog.h"
+#include "MRCommandLoop.h"
+#include "backends/imgui_impl_opengl3.h"
 
 namespace MR
 {
 
-static void loadFontChecked( const char* filename, float size_pixels, const ImFontConfig* font_cfg = nullptr, const ImWchar* glyph_ranges = nullptr )
+static ImFont* loadFontChecked( const char* filename, float size_pixels, const ImFontConfig* font_cfg = nullptr, const ImWchar* glyph_ranges = nullptr )
 {
-    if ( !ImGui::GetIO().Fonts->AddFontFromFileTTF( filename, size_pixels, font_cfg, glyph_ranges ) )
+    auto font = ImGui::GetIO().Fonts->AddFontFromFileTTF( filename, size_pixels, font_cfg, glyph_ranges );
+    if ( !font )
     {
         assert( false && "Failed to load font!" );
         spdlog::error( "Failed to load font from `{}`.", filename );
 
-        ImGui::GetIO().Fonts->AddFontFromMemoryCompressedTTF( droid_sans_compressed_data,
+        font =ImGui::GetIO().Fonts->AddFontFromMemoryCompressedTTF( droid_sans_compressed_data,
```

Comment:
Will it be loaded multiple times if multiple fonts fail to load?

Reply:
Yes, it will be loaded for each failed font